x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -392,17 +428,50 @@ func CalcTopicInitialRegret(regrets []alloraMath.Dec, epsilon alloraMath.Dec, pN
 		return alloraMath.ZeroDec(), err
 	}
 
-	minimumRegret := alloraMath.ZeroDec()
-	for i, regret := range regrets {
-		if i == 0 || regret.Lt(minimumRegret) {
-			minimumRegret = regret
-		}
+	// Calculate percentile
+	percentile, err := alloraMath.GetQuantileOfDecs(regrets, alloraMath.MustNewDecFromString("0.25"))
+	if err != nil {
+		return alloraMath.ZeroDec(), err
 	}
 
-	initialRegret, err := minimumRegret.Add(offSetTimesDenominator)
+	initialRegret, err := percentile.Add(offSetTimesDenominator)
 	if err != nil {
 		return alloraMath.ZeroDec(), err
 	}
 
 	return initialRegret, nil
 }
+
+// Determine if a worker is an experienced worker by checking
+// the inclusions count is greater than 1/alpha_regret
+func isExperiencedWorker(
```

Comment:
can we write separate isExperiencedForecaster and isExperiencedInferer functions

I am afraid this actorType enum is a footgun waiting to happen